piqueserver/networkdict.py

```diff
@@ -7,19 +7,18 @@ def get_network(cidr):
     try:
```

Comment:
Do we even need get_network? Why is it caching stuff?

Reply:
ip_network isn't an expensive call, we can remove it.

Reply:
What I thought might be going on is that it is constanty recreating these when syncing bans